rsa-cryptosystem-signature/index17.js

```diff
@@ -0,0 +1,32 @@
+const firstPrime = 2;
+const secondPrime = 5;
+const N = firstPrime * secondPrime;
+const phiOfN = 0;
+let publicKey = 0;
+
+function hashTheMessage(message) {
+  let hashValue = 0;
+  for (let i = 0, msgLength = message.length; i < msgLength; ++i) {
+    hashValue += message.charCodeAt(i);
+  }
+  return hashValue;
+}
+
+/*
+If any integer till the `smallerNum` divide both the numbers then they are `not coprime`.
```

Comment:
This might need to be broken down into two steps so we can better explain/review the remainder operator. Did my best to suggest some edits for now. Also, I'm sure my example code can be improved. Please feel free to change it in a future draft:

Divide `smallerNum` and `largerNum` by each of the numbers in the loop.
If `smallerNum` and `largerNum` can be divided evenly by any of those numbers, they are not co-prime.
Use an `if` statement to check if `smallerNum` and `largerNum` can be divided by `i` evenly.
Remember that the remainder (`%`)  operator can be used to check if one number is divisible by another:
```
const divisible = 9;
const notDivisible = 5;

console.log(divisible % 3); // 0
console.log(notDivisible % 2); // 1
```

Reply:
@scissorsneedfoodtoo This lesson is at the end part of the curriculum so, I am expecting campers to understand the remainder operator.

Reply:
@vkWeb, that's a good point. You're right that they should be familiar with this already, assuming they follow the curriculum. I was thinking more along the lines of having them check smallerNum in the first lesson, then write the check for biggerNum, but it should be fine to have them write both in the same step.

Can you think of another hint or example that doesn't give away so much of the answer? I'm all for providing the formula if it's complicated like some of the ones below, but I'm sure we can think of a way to guide the learners through this step.

Reply:
@scissorsneedfoodtoo Writing both the steps together will be the right way to go.

Use an if statement to check whether smallerNum and largerNum can be divided by i evenly. If i divides both the numbers then they aren't coprime.

☝️ I have just refined your description. How does it look?